pytorch_pfn_extras/nn/modules/lazy.py

```diff
@@ -90,24 +90,36 @@ def _lazy_load_hook(
         for the details of the hook specification.
         """
         for name in self.lazy_buffer_names:
-            # Avoid shape mismatch error when loading an initialized buffer
-            # onto an uninitialized module instance.
-            self.register_buffer(name, state_dict[prefix + name])
+            key = prefix + name
```

Comment:
I don't think it's necessary to take care of buffers.

Reply:
I agree as the modules with uninitialized buffers also have uninitialized parameters.